test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -321,6 +324,13 @@ def then_i_see_the_results_page_for_level_two_with_changing_restriction_levels
     assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.level_two.changing_alert_level", area: area))
   end
 
+  def then_i_see_details_of_christmas_rules
+    assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.christmas_rules.heading"))
+    assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.christmas_rules.body"))
+    assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.christmas_rules.guidance.label"))
+    assert page.has_css?("a[href='#{I18n.t('coronavirus_local_restrictions.results.christmas_rules.guidance.link')}']")
```

Comment:
It seems at odds with the rest of this test file to assert for every single translation phrase. Looking at [no information](https://github.com/alphagov/collections/blob/4a6e24b8ec9c5606f0d09d82dae52b3443beb20a/test/integration/coronavirus_local_restrictions_test.rb#L310) a precedent would be testing just the header.

Reply:
Make sense - I'll get these additional checks removed

Reply:
Done